packages/harmony/src/components/popup/Popup.tsx

```diff
@@ -55,15 +55,17 @@ const getComputedOrigins = (
 
   let containerWidth, containerHeight
   if (containerRef && containerRef.current) {
-    const containerRect = containerRef.current.getBoundingClientRect()
```

Comment:
is there a reason not to call `getBoundingClientRect` once and re-use the result?

Reply:
good q, this was auto-lint, looks like it removed something that wasn't needed. going to bring this back in case it is intentional though.